cmd/root_calibnet.go

```diff
@@ -18,6 +18,11 @@ limitations under the License.
 */
 package cmd
 
+import (
+	"github.com/glifio/go-pools/constants"
+)
+
 func init() {
+	chainID = constants.CalibnetChainID
```

Comment:
This definitely runs _after_ root.go? Just want to make sure root_calibnet overwrites `chainID` in `root.go` if the build flags are there

Reply:
Yes, tested using the [`glif info` cmd](https://github.com/glifio/cli/pull/58/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR46)